tools/build_release.py

```diff
@@ -9,7 +9,7 @@
 pp = pprint.PrettyPrinter(indent=4, width=100)
 
 default_python_cmd = ['python']
-drivers_to_upload = ['nidcpower', 'nidigital', 'nidmm', 'niswitch', 'nimodinst', 'nifgen', 'niscope', 'nise', 'nitclk']
+drivers_to_upload = ['nirfsg', 'nidcpower', 'nidigital', 'nidmm', 'niswitch', 'nimodinst', 'nifgen', 'niscope', 'nise', 'nitclk']
```

Comment:
Can you revert this change? We have a separate PR for the tooling changes.

Reply:
If changes to support RF are needed to the tooling, they should wait until after the tooling changes go in. We won't release nirfsg without them, anyway.